.travis.yml

```diff
@@ -5,7 +5,7 @@ rvm:
 services:
 - redis-server
 script:
-- bundle exec rspec
+- bundle exec rspec --format d
```

Comment:
Yeah. Some of the integration tests can be a little timing-dependent / flaky (eg. forking ancillary processes/servers), so it's hugely useful to have more detailed traces (in the CI logs).

Should all be fixed now, but I'd rather keep it in if you don't mind?